build.sbt

```diff
@@ -25,6 +26,20 @@ val javaDslApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-
 val javaApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-java/api/"
 val javaAndroidApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-android/api/"
 
+// GPG
+PgpKeys.gpgCommand in Global := "gpg2"
```

Comment:
Now that I'm thinking about it.. GPG is quite platform specific. Maybe we should not configure it at all and let the build use the defaults from `sbt-pgp` plugin. I think that should work for the CI.